enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -191,6 +191,7 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
           .map(DependencyNode::getArtifact)
           .filter(Objects::nonNull)
           .map(Artifact::getFile)
+          .filter(Objects::nonNull) // The root project's file is null if it's not installed locally
```

Comment:
Do we not check the root project in that case?

This could probably use a test if it caused a bug somewhere.